msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md

```diff
@@ -62,16 +62,15 @@ The `context` property determines what must be shown when a user invokes an app
 | **meetingStage** | An app from the meetingSidePanel can be shared to the meeting stage. |
 
 > [!NOTE]
-> `Context` property is currently not supported on mobile clients.
+> * Context properties **meetingDetailsTab** and **meetingChatTab** are the same for mobile. You can specify either of these to ensure apps work in mobile.
+> * **meetingStage** is not supported on mobile.
 
 After you enable your app for Teams meetings, you must configure your app before a meeting, during a meeting, and after a meeting.
 
 ## Configure your app for meeting scenarios
 
 > [!NOTE]
 > * For your app to be visible in the tab gallery, it must support configurable tabs and the group chat scope.
-> * Mobile clients support tabs only in pre and post meeting stages.
-> * The in-meeting experiences that is in-meeting dialog box and tab is currently not supported on mobile clients. For more information, see [guidance for tabs on mobile](../tabs/design/tabs-mobile.md) while creating your tabs for mobile.
```

Comment:
Currently, the in-meeting experiences, such as in-meeting dialog box and tab is not supported on mobile clients.

Reply:
These notes have been removed.